tests/robot-tests/tests/libs/common.robot

```diff
@@ -1014,3 +1013,20 @@ user takes html snapshot of element
 
 user waits for caches to expire
     sleep    %{WAIT_CACHE_EXPIRY}
+
+user wait for option to be available and select it
+    [Arguments]  ${dropdown_locator}  ${option_text}  ${timeout}=%{TIMEOUT}
+    wait until keyword succeeds  ${timeout}  1s  check option exist in dropdown  ${dropdown_locator}  ${option_text}
```

Comment:
Not seen `wait until keyword succeeds` - looks like that could help us a lot with flaky tests :)

Reply:
yeah I have seen few people talking in the stack overflow discussing about wait until keyword succeeds keyword. Here is the one of the link - https://stackoverflow.com/questions/51501685/is-there-any-keyword-to-serve-as-wait-until-keywords-succeed

Reply:
yeah it reduces flakiness a lot